content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,333 @@
+scan is to obtain information about possible vulnerabilities and their
+prevention in the container image, such as a
+[Common Vulnerability Scoring System (CVSS)](https://en.wikipedia.org/wiki/Common_Vulnerability_Scoring_System)
+score. If the result of the image scans is combined with the pipeline
+compliance rules, only properly patched container images will end up in
+Production.
+
+## Admission controllers
+
+- [ ] An appropriate selection of admission controllers is enabled.
+- [ ] A pod security policy is enforced by the Pod Security Admission or/and a
+  webhook admission controller.
+- [ ] The admission chain plugins and webhooks are securly configured.
+
+Admission controllers can help to improve the security of the cluster. However,
+they can present risks themselves as they extend the API server and
+[should be properly secured](/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/).
+
+The following list present a number of admission controllers that could be
+considered to enhance the security posture of your cluster and application. It
+includes controllers that may be referenced in other parts of this document:
+
+[`AlwaysPullImages`](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages)
+: Enforces the latest version of a tagged image is used, and that the deployer
+has permissions to use the image.
+
+[`CertificateApproval`](/docs/reference/access-authn-authz/admission-controllers/#certificateapproval)
+: Performs additional authorization checks to ensure the approving user has
+permission to approve certificate request.
+
+[`CertificateSigning`](/docs/reference/access-authn-authz/admission-controllers/#certificatesigning)
+: Performs an additional authorization checks to ensure the signing user has
+permission to sign certificate requests.
+
+[`CertificateSubjectRestriction`](/docs/reference/access-authn-authz/admission-controllers/#certificatesubjectrestriction)
+: Rejects any certificate request that specifies a 'group' (or 'organization
+attribute') of `system:masters`.
+
+[`DenyServiceExternalIPs`](/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips)
+: Prevents usage of `externalIPs` services.
+
+[`EventRateLimit`](/docs/reference/access-authn-authz/admission-controllers/#eventratelimit)
+: Rate limits adding new Events to the API server.
+
+[`ImagePolicyWebhook`](/docs/reference/access-authn-authz/admission-controllers/#imagepolicywebhook)
+: Allows enforcing additional controls for images through webhooks.
+
+[`MutatingAdmissionWebhook`](/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)
+: Allows the use of custom controllers through webhooks, these controllers may
+mutate requests that it reviews.
+
+[`NodeRestriction`](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+: Restricts kubelet's permissions to only modify the pods API resources they own or
```

Comment:
This is a bit confusing. Do we mean to say that this restricts kubelet's permissions to only modify the pods' API resources they own or the pods' resources themselves?
```suggestion
: Restricts kubelet's permissions to only modify the pods API resources they own or
```

Reply:
Ok! https://github.com/kubernetes/website/pull/33992/commits/791a833bb17731781e8ba4bc875422e8d56b4277.